econml/tests/test_automated_ml.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import unittest
+import pytest
+from sklearn.linear_model import LinearRegression, Lasso, \
+    LogisticRegression
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import OneHotEncoder, FunctionTransformer, \
+    PolynomialFeatures
+from sklearn.model_selection import KFold
+from econml.dml import *
+from econml.metalearners import *
+from econml.automated_ml import *
+from econml.drlearner import DRLearner
+import numpy as np
+from econml.utilities import shape, hstack, vstack, reshape, \
+    cross_product
+from econml.inference import BootstrapInference
+from contextlib import ExitStack
+from sklearn.ensemble import RandomForestRegressor, \
+    GradientBoostingRegressor, GradientBoostingClassifier
+import itertools
+from econml.sklearn_extensions.linear_model import WeightedLasso
+from econml.automated_ml import addAutomatedML
+from econml.tests.test_statsmodels import _summarize
+import econml.tests.utilities  # bugfix for assertWarns
+import copy
+import logging
+from econml.data.dgps import ihdp_surface_B
+from azureml.train.automl.exceptions import ClientException
+from azureml.core.authentication import AzureCliAuthentication
+import os
+
+AutomatedTLearner = addAutomatedML(TLearner)
+AutomatedSLearner = addAutomatedML(SLearner)
+AutomatedXLearner = addAutomatedML(XLearner)
+AutomatedDomainAdaptationLearner = \
+    addAutomatedML(DomainAdaptationLearner)
```

Comment:
These don't need to be split across lines, and would probably look cleaner without this.